spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java

```diff
@@ -755,7 +755,7 @@ protected void invokeBeanFactoryPostProcessors(ConfigurableListableBeanFactory b
 		PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(beanFactory, getBeanFactoryPostProcessors());
 
 		// Detect a LoadTimeWeaver and prepare for weaving, if found in the meantime
-		// (e.g. through an @Bean method registered by ConfigurationClassPostProcessor)
+		// (e.g. through a @Bean method registered by ConfigurationClassPostProcessor)
```

Comment:
When read as "at Bean", the use of "an" is correct.

Thus, please revert this change.